misc/extension_api_validation/4.2-stable.expected

```diff
@@ -51,3 +51,12 @@ Validate extension JSON: Error: Field 'classes/RenderingDevice/methods/texture_u
 Barrier arguments have been removed from all relevant functions as they're no longer required.
 Draw and compute list overlap no longer needs to be specified.
 Initial and final actions have been simplified into fewer options.
+
+GH-19564
+--------
+Validate extension JSON: Error: Field 'classes/TileMap/methods/get_collision_visibility_mode': is_const changed value in new API, from false to true.
+Validate extension JSON: Error: Field 'classes/TileMap/methods/get_navigation_visibility_mode': is_const changed value in new API, from false to true.
+Validate extension JSON: Error: Hash changed for 'classes/TileMap/methods/get_collision_visibility_mode', from 78C71571 to 6526718C. This means that the function has changed and no compatibility function was provided.
```

Comment:
I think a non-const compatibility method needs to be bound for this

CC @dsnopek

Reply:
Yeah, I think `const`-ness is part of the hash. (And we should never be ignoring any of the "hash changed" messages)

Reply:
Should be fixed now.